### ISSUE TITLE
Accessibility problem VoiceOver not announcing some actions taken

### ISSUE THREAD

**Issue by SRonald675** [erictli/scratch#171]
Voicer is not announcing when selected text is deleted.
  
Should announce the deleted text in a lower pitch depending on VoiceOver settings

No audio feedback upon pressing return
  Should announce "new line"
